### PR TITLE
Added Basic CSS and HTML 

### PR DESCRIPTION
**Pull request by Lsimmons98**
Added CSS and HTML file

### CODE REVIEW

**Comment by Lsimmons98** [comment]
Screenshot 2024-05-28 at 2 29 40 PM

File: README.md
Comment (stephenmckeon) [inline README.md:1]: These seem like notes, are you sure you want them included in this PR?

File: exercises.json [hunk 1]
```diff
@@ -0,0 +1,329 @@
+{
+  "exercises":
+  [
```
Comment (stephenmckeon) [inline exercises.json]:
```suggestion
  "exercises": [
```

File: exercises.json [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+    "exercise_id": 38,
+    "exercise_name": "Walking Lunges",
+    "muscle_group": "Legs",
+    "sub_group": "Glutes",
+    "category": "Compound",
+    "image": "https://images.squarespace-cdn.com/content/v1/537b0649e4b0f6c6b6877a35/1607948016856-YTYC2F8TRPJE2UTXLOM5/quads-glutes-hamstrings-calves.jpg",
+    "instructions": "Stand with feet together, step forward with one foot and lower your body until both knees are bent at 90-degree angles. Push back up to the starting position and repeat with the other leg."
+  }
+      ]
+    }
```
Comment (stephenmckeon) [inline exercises.json]:
```suggestion
  }]
}
```

**Review by stephenmckeon — COMMENTED** [review]
Looks good, just clean up the README changes and the formatting in your DB file. Also, the picture should be included in the PR description going forward, and the description itself is a bit lacking in its current state. Give it some more thought--a PR description should include a brief summary of the changes made, their purpose, and any relevant context or references to related issues or tasks.

**Pull request merged**